File: solutions/005/005.js
function gcd(a, b) {
  while (b !== 0) {
    c = b
    b = a % b
    a = c
  }
  return a
}

function lcm(a, b) {
  return a * b / gcd(a, b)
}

let result = 1
for (let i = 1; i < 20; i++)
  result = lcm(result, i)
console.log(result)

File: solutions/005/005.py
def lcm(a, b):
    return a * b / gcd(a, b)


def gcd(a, b):
    while b != 0:
        a, b = b, a % b
    return a

result = 1
for i in range(1, 20 + 1):
    result = lcm(result, i)
print(int(result))
